test/powershell/Modules/PSReadLine/PSReadLine.Tests.ps1

```diff
@@ -42,6 +42,12 @@ Describe "PSReadLine" -tags "CI" {
         (Get-PSReadLineKeyHandler | where { $_.Key -ceq "Alt+B" }).Function | Should Be SelectBackwardWord
     }
 
+    It "Should ignore case when using Function binding" {
+        Set-PSReadlineOption -EditMode Windows
+        Set-PSReadlineKeyHandler "Ctrl+k" -Function yank # NOTE: lower case function name
+        (Get-PSReadlineKeyHandler | Where { $_.Key -ceq "Ctrl+k"}).Function | Should Be Yank
```

Comment:
Please use quotation for string constant - "Yank".
The test is case-sensitive so we should use `BeExactly` and make the comment clearer:
```powershell
# Attention! The test is case-sensitive! Don't change the cases!
$custom_name = "yank" ; $real_name = "Yank"

Set-PSReadlineOption -EditMode Windows
Set-PSReadlineKeyHandler "Ctrl+k" -Function $custom_name
(Get-PSReadlineKeyHandler | Where { $_.Key -ceq "Ctrl+k"}).Function | Should BeExactly $real_name
```